src/components/Result/ResultHeader.vue

```diff
@@ -45,4 +48,9 @@ export default {
 .skewed-top-wrapper {
 	clip-path: ellipse(95% 100% at 50% 0%);
 }
+
+@media (max-width: 768px) {
+	.skewed-top-wrapper {
```

Comment:
please use the scss ampersand syntax and merge it with the declaration of skewed-top-wrapper above.

Reply:
![image](https://user-images.githubusercontent.com/32281312/109690148-519cd980-7b86-11eb-867c-0618abca0244.png)

I don't understand how I should use ampersand.

Reply:
https://css-tricks.com/the-sass-ampersand/

Reply:
I tried @screen md also @screen-md and other options but it doesn't work so I use @media and &.